hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1641,7 +1641,7 @@ void optimizeARM(Vunit& unit, const Abi& abi, bool regalloc) {
   simplify(unit);
 
   lowerForARM(unit);
-
+  reuseImmq(unit);
```

Comment:
Actually, I'm not convinced it matters one way or the other.  The foldIms() pass is detecting the case
where the immediate can be pushed into the instruction.  This pass is just checking for immediates built up 
from multiple instructions.  @mxw, @cmuellner